### PR TITLE
View files Menu Fix

### LINKED ISSUE #747: Bug in View Files
Bug in the file selection

### Steps to reproduce
* Select Any file 
* Now click `Select All` menu.

### Expected behaviour
All files get selected.

### Actual behaviour
Files get Deselected.

### PR DESCRIPTION
**Pull request by sunnyjain**
# Description
ezgif com-video-to-gif

Fixes #747 and #763 

## Type of change
Just put an x in the [] which are valid.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by vyankatesh24** [comment]
@sunnyjain 
It is not working properly. It should work like a toggle button, 'Select All' and 'Unselect All'. It was working like this only but it used to `Unselect` the first selected and then it use to 'Select All'

**Comment by sunnyjain** [comment]
So you want to keep the checkbox? because when we select all the merge button appears and then the select all button goes inside the option.
what we can do is when all the files are selected, we can make a unselect all button appear besides merge. works?

**Comment by vyankatesh24** [comment]
Sure :+1:

**Comment by vyankatesh24** [comment]
@sunnyjain 
Are you GSSoC participant?

**Comment by sunnyjain** [comment]
> @sunnyjain
> Are you GSSoC participant?

nope.

File: app/src/main/res/menu/activity_view_files_actions.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
         app:showAsAction="ifRoom" />
     <item
         android:id="@+id/select_all"
-        android:icon="@drawable/ic_check_box_outline_blank_24dp"
+        android:icon="@drawable/ic_check_box_24dp"
```
[the remark this answers is not in the document]
Comment (sunnyjain) [reply app/src/main/res/menu/activity_view_files_actions.xml]: I believe the icons are correct there, as checked checkbox denotes 'Select All' and unchecked denotes 'Unselect all'.

File: app/src/main/java/swati4star/createpdf/fragment/ViewFilesFragment.java [hunk 2]
```diff
@@ -349,30 +356,23 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
 
     @Override
     public void isSelected(Boolean isSelected, int countFiles) {
+        mCountFiles = countFiles;
+        updateToolbar();
+    }
+
+    public void updateToolbar() {
```
[the remark this answers is not in the document]
Comment (sunnyjain) [reply app/src/main/java/swati4star/createpdf/fragment/ViewFilesFragment.java]: Sure!

File: app/src/main/res/menu/activity_view_files_actions.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (vyankatesh24) [reply app/src/main/res/menu/activity_view_files_actions.xml]: > I believe the icons are correct there, as checked checkbox denotes 'Select All' and unchecked denotes 'Unselect all'.

Just as single checkbox, `unchecked` means it is `not selected` and `checked` means it is `selected` 
Same here, Unchecked denotes `All are not selected` and checked denotes `All are selected`  .

[the remark this answers is not in the document]
Comment (sunnyjain) [reply app/src/main/res/menu/activity_view_files_actions.xml]: Aright, makes sense.

**Comment by vyankatesh24** [comment]
Congratulations @sunnyjain on your very first contribution to `Images-to-PDF` :1st_place_medal:

**Pull request merged**